go/ethclient/mock_client.go

```diff
@@ -0,0 +1,25 @@
+package ethclient
```

Comment:
I think this should live over in `/integration`, since it's just used for tests.

Reply:
It will also be used to connect the real eth node. Different implementation.